src/nexus.cpp

```diff
@@ -358,7 +358,8 @@ void Nexus::updateWindowsArg(QWindow* closedWindow)
         QAction* action = windowActions->addAction(windowList[i]->title());
         action->setCheckable(true);
         action->setChecked(windowList[i] == activeWindow);
-        connect(action, &QAction::triggered, [=] { onOpenWindow(windowList[i]); });
+        connect(action, &QAction::triggered, this,
+                [this, &windowList, i] { onOpenWindow(windowList[i]); });
```

Comment:
Wouldn't the reference to `windowList` be dangling? Maybe something like `window = windowList[i]` (since `QWindowList` is a `QList<Window *>`, apparently) would be better?

Reply:
Done.